srcpkgs/retroarch/template

```diff
@@ -1,35 +1,36 @@
 # Template file for 'retroarch'
 pkgname=retroarch
-version=1.8.4
+version=1.9.0
 revision=1
 wrksrc="RetroArch-$version"
 build_style=configure
-configure_args="--prefix=/usr --sysconfdir=/etc --enable-udev
- --enable-networking
- $(vopt_enable ffmpeg) $(vopt_enable jack) $(vopt_enable pulseaudio pulse)
- $(vopt_enable sdl2) $(vopt_enable x11) $(vopt_enable vulkan)"
+configure_args="--prefix=/usr --sysconfdir=/etc --enable-networking
+ --enable-udev --disable-builtinflac --disable-builtinglslang
+ --disable-builtinmbedtls --disable-builtinminiupnpc --disable-builtinzlib
+ $(vopt_enable ffmpeg) $(vopt_enable glslang) $(vopt_enable jack)
+ $(vopt_enable pulseaudio pulse) $(vopt_enable sdl2) $(vopt_enable vulkan)
+ $(vopt_enable x11)"
 conf_files="/etc/retroarch.cfg"
 hostmakedepends="pkg-config python unzip"
-makedepends="zlib-devel libxml2-devel freetype-devel libxkbcommon-devel
- alsa-lib-devel eudev-libudev-devel $(vopt_if opengl MesaLib-devel)
- $(vopt_if pulseaudio pulseaudio-devel) $(vopt_if jack jack-devel)
- $(vopt_if ffmpeg ffmpeg-devel) $(vopt_if sdl2 SDL2-devel)
- $(vopt_if x11 'libXext-devel libXinerama-devel libXv-devel libXxf86vm-devel')
- $(vopt_if vulkan 'vulkan-loader')
- $(vopt_if qt5 'qt5-devel')"
+makedepends="alsa-lib-devel eudev-libudev-devel freetype-devel libflac-devel
+ libxkbcommon-devel libxml2-devel mbedtls-devel miniupnpc-devel zlib-devel
+ $(vopt_if ffmpeg ffmpeg-devel) $(vopt_if glslang 'glslang-devel SPIRV-Tools-devel')
+ $(vopt_if jack jack-devel) $(vopt_if opengl MesaLib-devel) $(vopt_if pulseaudio pulseaudio-devel)
+ $(vopt_if qt5 qt5-devel) $(vopt_if sdl2 SDL2-devel) $(vopt_if vulkan vulkan-loader)
+ $(vopt_if x11 'libXext-devel libXinerama-devel libXv-devel libXxf86vm-devel')"
 depends="$(vopt_if vulkan 'vulkan-loader')"
 short_desc="Official reference frontend for the libretro API"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="http://www.libretro.com/"
 distfiles="https://github.com/libretro/RetroArch/archive/v$version.tar.gz"
-checksum=027e9eb243488da1b8532116eab2a79932fcf2dcb96bcc4be9a12b329a2a9c52
-build_options="ffmpeg opengl jack pulseaudio sdl2 x11 vulkan qt5"
+checksum=39d7ad9298c9487fb4848cd6a257c23a95ee85f83c75302f97bcc4797f154b6a
+build_options="ffmpeg glslang jack opengl pulseaudio qt5 sdl2 vulkan x11"
 
 build_options_default="ffmpeg"
 
 case "$XBPS_TARGET_MACHINE" in
-	i686*|x86_64*|ppc64*|aarch64*) build_options_default+=" opengl pulseaudio sdl2 x11 vulkan";;
+	i686*|x86_64*|ppc64*|aarch64*) build_options_default+=" glslang opengl pulseaudio sdl2 vulkan x11";;
 	armv[67]*)
 		makedepends+=" rpi-userland-devel"
 		LDFLAGS="-L${XBPS_CROSS_BASE}/opt/vc/lib -lbcm_host"
```

Comment:
Since we are at this, @q66 do you think we can remove this whole case and just use the same options for all platforms?

The other option is making `rpi` a build option, not set by default, to build retropie with `rpi-userland-devel`.

Reply:
yeah, just drop it

we shouldn't have any unmarked rpi-specific packages